CMakeLists.txt

```diff
@@ -1,16 +1,18 @@
 cmake_minimum_required(VERSION 3.19)
 project(my_application LANGUAGES CXX)
 
-# Note: Use find_package(SixtyFPS) instead of the following three commands, if you prefer the package
-# approach.
-include(FetchContent)
-FetchContent_Declare(
+find_package(SixtyFPS)
```

Comment:
should we use QUIET here or not?

By default, cmake will output this warning if the package was not found:

```
CMake Warning at CMakeLists.txt:4 (find_package):
  By not providing "FindSixtyFPS.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "SixtyFPS",
  but CMake did not find one.

  Could not find a package configuration file provided by "SixtyFPS" with any
  of the following names:

    SixtyFPSConfig.cmake
    sixtyfps-config.cmake

  Add the installation prefix of "SixtyFPS" to CMAKE_PREFIX_PATH or set
  "SixtyFPS_DIR" to a directory containing one of the above files.  If
  "SixtyFPS" provides a separate development package or SDK, be sure it has
  been installed.
```

So we have three choices
 1. leave it like that
 2. use QUIET,  so it doesn't show any message and just use SixtyFPS from git
 3. use QUIET, but still add a `message("No binary packages of SixtyFPS found, cmake will download and compile packages from git. Checkout https://sixtyfps.io/docs/cpp/install.html for more info")` (or something like that)

Reply:
Ah, you had the same idea at the same time hehe

Reply:
My preference is (3).

Ultimately I think what's really important is that the cmake configure output gives an indication what happened / is going to happen. So with (3) it would tell what's going to happen when the package could not be found in the cmake module search path. But perhaps we should also write an explicit message *when* it was found.